src/libcore/ops.rs

```diff
@@ -1539,6 +1539,10 @@ impl<Idx: PartialOrd<Idx>> Range<Idx> {
 ///
 /// See the [`contains()`](#method.contains) method for its characterization.
 ///
+/// Note: No overflow checking is done for the iterator implementation; if you
+/// use an integer range and the integer overflows, it will panic in debug mode
+/// and create an endless loop in release mode.
```

Comment:
Ah right. I guess I'm basically thinking we want to word this in such a way so that we're not committing to anything, just noting the current behavior.